conda_recipe/darts/meta.yaml

```diff
@@ -1,12 +1,12 @@
 {% set data = load_setup_py_data() %}
 
 package:
-  name: "u8timeseries"
+  name: "darts"
```

Comment:
Are we publishing to conda under name darts?

Reply:
I don't think it's planned yet.